docs/FAQ.md

```diff
@@ -22,7 +22,9 @@ We hope to see Backstage become the infrastructure standard everywhere. When we
 
 Yes, we've already started releasing open source versions of some of the plugins we use here, and we'll continue to do so. [Plugins](https://github.com/spotify/faq#what-is-a-plugin-in-backstage) are the building blocks of functionality in Backstage. We have over 120 plugins inside Spotify — many of those are specialized for our use, so will remain internal and proprietary to us. But we estimate that about a third of our existing plugins make good open source candidates. (And we'll probably end up writing some brand new ones, too.)
 ​
+
```

Comment:
These (probably unintended?) are causing merge collisions.

Reply:
Damn, not prettier again.. On it!)